### PR TITLE
Creation of the website skeleton

### PR DESCRIPTION
**Pull request by ThibaultBardinetLanglois**

### CODE REVIEW

**Review by ced37 — DISMISSED** [review]

**Review by StephaneSauvaget — DISMISSED** [review]
Ok for me

File: src/Controller/ClientController.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+* @Route("/maître-d'ouvrage", name="client_")
```
Comment (RomainClair) [inline src/Controller/ClientController.php]: you probably should use more simple/small URLs
/moa/list may be ?

File: src/Controller/ProfessionnalController.php [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+* @Route("/maître-d'oeuvre", name="professionnal_")
```
Comment (RomainClair) [inline src/Controller/ProfessionnalController.php]: Same here
/moe,  /pro ?

Comment (StephaneSauvaget) [reply src/Controller/ProfessionnalController.php]: Nice clever tricks ! thanks

**Review by ced37 — APPROVED** [review]

**Review by RomainClair — APPROVED** [review]

**Pull request merged**